### ISSUE TITLE
model-55000.ckpt

### ISSUE THREAD

**Issue by SHYalien** [YangZeyu95/unofficial-implement-of-openpose#1]
Bro,can you share the model-55000.ckpt file? Our own GPU capability is not enough. It takes about three weeks to get results. I look forward to your reply, thank you.

**Comment by YangZeyu95** [comment]
ok,  give me a second , i have update the code to allow restore fine-tuned vgg, so pull and just set the path ckpt in and the code would find the latest ckpt

**Comment by YangZeyu95** [comment]
checkpoint is uploaded 链接: https://pan.baidu.com/s/1bsPsoftgcdcIr4KwXu4Ujw 提取码: pn5f, if you want Google driver version please see readme

**Comment by SHYalien** [comment]
Thank you for sharing. Error running'python run.py -- checkpoint_path checkpoints/train/model-59000.ckpt'

'OP_REQUIRES failed at save_restore_tensor.cc:175: Data loss: Unable to open table file checkpoints/train/model-59000.ckpt: Data loss: not an sstable (bad magic number): perhaps your file is in a different file format and you need to use a different restore operator?'

**Comment by YangZeyu95** [comment]
You might have to download the latest code, i have changed run.py today.

**Comment by SHYalien** [comment]
Thank you, we have used the latest run.py, but still reported this error.

**Comment by SHYalien** [comment]
Is our version of TensorFlow inconsistent?

**Comment by YangZeyu95** [comment]
My TensorFlow version is 1.12. And what’s your checkpoint_path?  Path should be the folder include .ckpt rather than .ckpt file.

**Comment by SHYalien** [comment]
![2018-12-14_23-07-11](https://user-images.githubusercontent.com/32799122/50010970-ca3f0c00-fff5-11e8-90eb-2ffdfed787f9.png)
![2018-12-14_23-07-57](https://user-images.githubusercontent.com/32799122/50010971-ca3f0c00-fff5-11e8-9d5a-a73de8b18a2d.png)
![2018-12-14_23-08-34](https://user-images.githubusercontent.com/32799122/50010974-cad7a280-fff5-11e8-948b-16448f58dd69.png)
Thank you for your reply, the program has been able to run through, test the picture - ‘ski.jpg’ is quite strange, can you leave a contact (My WeChat: A3_CLOWN52), I look forward to your reply.

**Comment by YangZeyu95** [comment]
搜你微信用户不存在 我的yangzeyu0766

**Comment by YangZeyu95** [comment]
checkpoint file updated
